### PR TITLE
Friendly

### LINKED ISSUE #68: Add friendly-id to ft-forms
Currently, to get around the fact that the `show` action in a controller requires that an ID be entered into the URL bar (which isn't very human-readable) there's a [controller action](https://github.com/umts/ft-forms/blob/master/app/controllers/forms_controller.rb#L16-L20) simple called 'meet_and_greet'. Instead of doing something like that (which we'd have to do for any future forms added), we could use a gem called [friendly-id ](https://github.com/norman/friendly_id), in which you use a 'slug' attribute to define what shows up in the URL instead of the ID.
[the documentation ](http://norman.github.io/friendly_id/file.Guide.html)

### PR DESCRIPTION
**Pull request by artarazavi**
closes #68

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -44,8 +44,22 @@
...
+    t.integer  "sluggable_id",   limit: 4,   null: false
+    t.string   "sluggable_type", limit: 50
+    t.string   "scope",          limit: 255
+    t.datetime "created_at"
+  end
+
+  add_index "friendly_id_slugs", ["slug", "sluggable_type", "scope"], name: "index_friendly_id_slugs_on_slug_and_sluggable_type_and_scope", unique: true, using: :btree
+  add_index "friendly_id_slugs", ["slug", "sluggable_type"], name: "index_friendly_id_slugs_on_slug_and_sluggable_type", using: :btree
+  add_index "friendly_id_slugs", ["sluggable_id"], name: "index_friendly_id_slugs_on_sluggable_id", using: :btree
+  add_index "friendly_id_slugs", ["sluggable_type"], name: "index_friendly_id_slugs_on_sluggable_type", using: :btree
```
Comment (Anbranin) [inline db/schema.rb]: What's going on with all this? There is no migration that creates the table friendly_id_slugs, nor these indices. It looks like we aren't using that table anywhere, either.

File: spec/controllers/forms_controller_spec.rb [hunk 2]
```diff
@@ -49,7 +49,7 @@
       @form = create :form, name: 'Meet & Greet Request Form'
```
Comment (Anbranin) [inline spec/controllers/forms_controller_spec.rb]: Could you change the description here? We're no longer testing `meet_and_greet`

File: app/models/form.rb [hunk 3]
```diff
@@ -29,4 +31,8 @@ def draft_belonging_to(user)
   def draft_belonging_to?(user)
     draft_belonging_to(user).present?
   end
+
+  def form_name
+    [name.parameterize].join('-')
```
Comment (dfaulken) [inline app/models/form.rb]: Join an array of a single thing?

Comment (Anbranin) [inline app/models/form.rb]: parameterize already returns what we need. The joins is unnecessary here. I'll remove it.

**Pull request merged**